source/adapters/hip/program.cpp

```diff
@@ -495,3 +495,8 @@ UR_APIEXPORT ur_result_t UR_APICALL urProgramGetFunctionPointer(
 
   return Result;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urProgramGetGlobalVariablePointer(
+    ur_device_handle_t, ur_program_handle_t, const char *, size_t *, void **) {
+  return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;
```

Comment:
Would it be possible to also implement this for the HIP adapter? You should be able to pretty much share the same code across CUDA and HIP adapters

Reply:
Added a HIP implementation in the latest commit :+1: